skiko/src/commonMain/kotlin/org/jetbrains/skia/BreakIterator.kt

```diff
@@ -523,15 +527,30 @@ class BreakIterator internal constructor(ptr: NativePointer) : Managed(ptr, _Fin
     }
 }
 
+private fun withErrorGuard(message: String, block: InteropScope.(InteropPointer) -> NativePointer): NativePointer {
+    val errorCode = IntArray(1)
+    return interopScope {
+        val handle = toInterop(errorCode)
+        val res = block.invoke(this, handle)
+        handle.fromInterop(errorCode)
+        if (errorCode[0] > 0) {
+            throw RuntimeException("$message; ubrk_* operation failed with status ${errorCode}")
```

Comment:
Maybe better `"$message: operation failed with status ${errorCode}"` as code could be reused in other places as well.

Reply:
I agree - pushed